cmake/MSCCLPP.cmake

```diff
@@ -63,15 +73,16 @@ if(ENABLE_MSCCLPP)
         mscclpp_cmake_arg(HIP_COMPILER)
     
         download_project(PROJ                mscclpp_nccl
-                         GIT_REPOSITORY      https://github.com/microsoft/mscclpp.git
-                         GIT_TAG             1e82dd444fc1ed8b7add354eebaab8a94e67d5fc
+                         # GIT_REPOSITORY      https://github.com/microsoft/mscclpp.git
+                         # GIT_TAG             1e82dd444fc1ed8b7add354eebaab8a94e67d5fc
```

Comment:
I left this in as a comment to act as documentation of the commit hash, since it's more visible. But there's also the risk of them getting out of sync if the maintainer forgets to update it. Should it be removed?